myproject/templates/new_entry.jinja2

```diff
@@ -0,0 +1,15 @@
+{% extends "layout.jinja2" %}
+{% block body %}
+
+    <form>
+        Title:<br>
```

Comment:
In general, you should not need to use `<br/>` tags.  Having them in your HTML is a code smell to me that means you aren't paying enough attention to your stylesheets.